releasenotes/notes/jupyter_remove-e3767b490c8dcf14.yaml

```diff
@@ -0,0 +1,12 @@
+---
+features:
```

Comment:
This is not a feature it would have to be an upgrade note. Since it's a change that users will need to update they're notebooks after upgrading to the next terra version.